### PR TITLE
Fixes/refs #11572 - defaut spool directory has wrong context

### PR DESCRIPTION
**Pull request by rodehoed**
Please let me know if this oke

### CODE REVIEW

File: foreman-proxy.te [hunk 1]
```diff
@@ -67,6 +67,13 @@ gen_tunable(foreman_proxy_manage_puppetca, true)
 ## </desc>
 gen_tunable(foreman_proxy_use_sudo, true)
 
+## <desc>
+## <p>   
+## Support for Openscap plugin
```
Comment (domcleal) [inline foreman-proxy.te:72]: Nitpick, `OpenSCAP`

File: foreman-proxy.te [hunk 2]
```diff
@@ -67,6 +67,13 @@ gen_tunable(foreman_proxy_manage_puppetca, true)
...
+## </p>                       
+## </desc>                    
+gen_tunable(foreman_proxy_manage_openscap, true)
```
Comment (domcleal) [inline foreman-proxy.te:75]: Probably false, since the plugin's an optional extra.

**Comment by domcleal** [comment]
Should the spool directory (I don't know where it is precisely) be labelled differently, or specifically?  Most directories under /var/spool have specific contexts for me, rather than sharing one (which might allow a process like the proxy to add incoming/untrusted data to another service).

Regarding the commit message, it should just be "fixes #11572" - I've improved the warning message as it was unclear - sorry!

**Comment by rodehoed** [comment]
oke then I need some guidance. Should I create a foreman_proxy_var_spool_t context and a file label for /var/spool/foreman-proxy(/.*)?

**Comment by domcleal** [comment]
Yep, sounds good.

File: foreman-proxy.te [hunk 3]
```diff
@@ -274,3 +281,14 @@ tunable_policy(`foreman_proxy_manage_dhcp_isc', `
...
+
+tunable_policy(`foreman_proxy_manage_openscap', `
+    gen_require(`
+        type foreman_proxy_t;
+        type var_spool_t;
+    ')
+
+    # Allow foreman_proxy_t to var_spool_t
+    allow foreman_proxy_t var_spool_t:dir { write create add_name };
+    allow foreman_proxy_t var_spool_t:file { write create open };
```
Comment (lzap) [inline foreman-proxy.te:293]: Can you use manage_dirs_pattern and manage_files_pattern patterns instead?

**Comment by lzap** [comment]
Also make sure to update `-relabel` script to include the dir.

**Comment by lzap** [comment]
Ping @rodehoed ? Merging time, we have few nitpicks, can you incorporate them please?

**Comment by lzap** [comment]
I hope you are well @rodehoed, if you don't have time for this, just let us know and I will take over.

**Comment by rodehoed** [comment]
Sorry @lzap I somehow missed the previous update. Well currently I don't have a good environment to play/test it with it. So for me it's oke you take it over. If I can help with anything just let me know :+1:

**Pull request closed without merging**